org.opentosca.planbuilder.provphase.plugin.invoker/src/org/opentosca/planbuilder/provphase/plugin/invoker/bpel/handlers/ResourceHandler.java

```diff
@@ -54,6 +54,20 @@ public ResourceHandler() throws ParserConfigurationException {
 
     }
     
+    private String loadFragmentResourceAsString(final String fileName) throws IOException {
```

Comment:
Exactly the same two methods are already implemented in `BPELProcessFragments`. Therefore, we should make it public there and reuse it here or factor it out to a utility class.